src/Mount.cpp

```diff
@@ -2299,7 +1991,7 @@ float Mount::getSpeed(int direction) {
 //
 // This code tells the steppers to what location to move to, given the select right ascension and declination
 /////////////////////////////////
-void Mount::calculateRAandDECSteppers(float& targetRA, float& targetDEC) {
+void Mount::calculateRAandDECSteppers(DayTime const& ra, Declination const& dec, long& targetRASteps, long& targetDECSteps) const {
```

Comment:
with HAL refactoring we should stop using DayTime as type for RA position. We should create a DEC and RA model classes.

Reply:
Agreed. There is a Declination class, dunno why there isn't an RA equivalent.

Reply:
RA would have no extra functionality to DayTime. But we should probably just derive it for clarity.